### ISSUE TITLE
Excessively large input object causes stack overflow

### ISSUE THREAD

**Issue by bweir** [parallaxinc/OpenSpin#27]
Hi there,

I stumbled across this trying to compile a way-too-large LameStation image into an object. OpenSpin detects the error properly but does a core dump instead of exiting gracefully, which then triggers the OS error reporting mechanism (making it seem like something horrible just happened). Just thought I should let you know.

[Click here](https://gist.githubusercontent.com/bweir/f118a5801c5285050e63/raw/dd2d1ddc5ffde02a6248a22ce7f609adb6525dd4/gfx_Blit.spin) to download the offending file.

```
$ ../OpenSpin/build/openspin gfx_Blit.spin 
Propeller Spin/PASM Compiler 'OpenSpin' (c)2012-2016 Parallax Inc. DBA Parallax Semiconductor.
Version 1.00.78 Compiled on Mar 25 2016 17:21:58
Compiling...
gfx_Blit.spin
gfx_Blit.spin(525:740) : error : Object exceeds 128k (before distilling)
Line:
word    $0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$5555,$5555,$0000,$0000,$0000,$0000,$0000,$0000,$5555,$0000,$0000,$0000,$5555,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$0000,$5555,$0000,$5555,$0000,$5555,$0000,$5555,$0000,$5555,$0000,$5555,$0000,$0000,$0000,$5555,$5555,$5555,$5555,$5555,$5555,$5555,$5555,$5555,$5555,$5555,$0000,$0000,$0000,$0000,$5555,$0000,$0000,$5555,$5555,$0000,$0000,$0000 ' ������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������������
Offending Item: ,
*** stack smashing detected ***: ../OpenSpin/build/openspin terminated
Aborted (core dumped)
```

**Comment by reltham** [comment]
Fixed in version 1.00.79

**Comment by bweir** [comment]
Cool, can't wait to integrate this release!

On Mon, May 30, 2016 at 6:35 PM, Roy Eltham <notifications@github.com>
wrote:

> Fixed in version 1.00.79
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/parallaxinc/OpenSpin/issues/27#issuecomment-222574379>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AJEcGOZ_d9BWQqCrwrC-VM1o69NemSQMks5qG5BTgaJpZM4H5Hk6>
> .
>



-- 
*Brett Weir*, Founder


Engineering Made Awesome
310.245.1775  • brett@lamestation.com • lamestation.com

**Comment by reltham** [comment]
Fair warning, the code got significant reorganizing in the wrapper stuff.  Also, more is coming soon (this week/weekend).
